packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -13,10 +13,26 @@ function Popover({
   height,
   zIndex,
   a11yId,
+  hover,
```

Comment:
How will you control the value of this prop?

Desktop should keep the `hover` behaviour right?

Reply:
> and click will default

I would agree with you, but that would break existing implementations that are relying on hover - which was the original behavior. So `hover` should be `true`.

Agree? :)

Reply:
In this way, places that use Popover will use the click as a default, what changes is that. This change was made precisely because, as the previous implementation did not work on mobile (as shown in the video in which the bug was reported), and the hover is not an expected behavior of a Popover, but of a tooltip. 
Remembering that this was discussed with the team before it was implemented, but we can open it up again for discussions, no problem :)

Reply:
Let's apply the changes discussed offline and everything will work well